tests/forms_tests/tests/tests.py

```diff
@@ -87,6 +87,12 @@ def test_empty_queryset_return(self):
         self.assertIsInstance(form.cleaned_data['multi_choice'], models.query.QuerySet)
 
 
+class TestTicket25683(TestCase):
```

Comment:
Oh ok. I thought about it and didn't want to put it under `ModelFormCallableModelDefault` because this test case doesn't involve ModelForms in my test. But I'll move it.

Sorry that I've been slow responding to these messages.